src/screens/ImportSeedPhraseSheet.js

```diff
@@ -87,7 +87,7 @@ const ConfirmImportAlert = (onSuccess, navigate) =>
     buttons: [
       {
         onPress: () =>
-          navigate(Routes.OVERLAY_EXPANDED_ASSET_SCREEN, {
+          navigate(Routes.MODAL_SCREEN, {
```

Comment:
This change breaks multiwallet.   When this modal opens, it dismisses the ImportSeed sheet, so it can't run the callback anymore.

Reply:
@brunobar79 i dont think it actually does break anything! as part of this PR i created a new Screen called `ModalScreen` which takes over the responsibility for these lil overlay screens, now that our `ExpandedStates` are *sheets* and not modals

 i just tested it with `isMultiwalletAvailable` set to both `true` and `false` and i believe it works as expected!